### DISCUSSION TITLE
Error 404

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37215] [New to GitHub] [answered]
i uploaded my website to github repository to host but when i try to navigate to another page it showing error 404

- There's no mistake in <a> tags

Please help as soon as possible, thanks

![Screenshot_20221025-190221](https://user-images.githubusercontent.com/88527795/197787456-bb2b468b-206f-47eb-bb85-2282dca59de7.jpg)

**Comment by godnondsilva** [comment, marked as answer]
@ViperVikxey The issue is that the website you are hosting is the repository name "viperweb". This means that GitHub Pages will host your website on the URL: https://vipervikxey.github.io/viperweb/

Additionally, If you would like to host your website on the home URL, you must rename the repository to "yourusername.github.io". In your case, it would be "vipervikxey.github.io"

Because it's not on your default home URL, the links aren't working.
For example, the link https://vipervikxey.github.io/viperweb/pages/codes.html works.
The "viperweb" is not added by default...

I will edit my reply to make things more clear, you have two possible fixes:
1. Update the repository name as mentioned in the 2nd paragraph
2. Update the links in the a tags inside your application (The complete links)

**Reply by ghost** [reply]
Hey @godnondsilva i just tried all steps you told me above but i still facing same problem when i try to navigate to other page please help

Things i changed
- changer repo name to my username (not worked)
- tried to add full URL in a tag (not worked)

Please help if there's other options

**Reply by godnondsilva** [reply]
- for the first change, the repository name should be "username.github.io", not just the username
- full URL means "username.github.io/repo-name/...."

**Reply by godnondsilva** [reply]
https://github.com/godnondsilva/godnondsilva.github.io

here is my GitHub pages repository, you can cross check with this.

**Reply by ghost** [reply]
@godnondsilva thank you so much for your help and time, now my site it working properly

**Reply by godnondsilva** [reply]
Glad to hear that! 😄

**Reply by coderzillla** [reply]
> https://github.com/godnondsilva/godnondsilva.github.io

hey I typed that in but with by name coderzilla instead of godnondsilva but i got sent to the other 404 with the star wars stuff

**Reply by godnondsilva** [reply]
@coderzillla have you deployed the repository to GitHub pages?

**Reply by coderzillla** [reply]
no i just copied and pasted yours with my name instead

**Reply by coderzillla** [reply]
never mind i was typing my name wrong the whole time with 2 L instead of 3

**Reply by godnondsilva** [reply]
ah, nice!

**Reply by ilovealbasit** [reply]
794834

**Comment by 0981901322** [comment]
> 我將我的網站上傳到 github 存儲庫進行託管，但是當我嘗試導航到另一個頁面時，它顯示錯誤 404
> 
> * 標籤沒有錯誤
> 
> 請盡快幫忙，謝謝
> 
> ![截圖_20221025-190221](https://user-images.githubusercontent.com/88527795/197787456-bb2b468b-206f-47eb-bb85-2282dca59de7.jpg)
